bot/exts/utils/subscription.py

```diff
@@ -0,0 +1,152 @@
+from bot.bot import Bot
+from bot.constants import Colours, Emojis, Roles
+from discord import Embed, Role
+from discord.ext.commands import Cog, Context, group
+
+
+class Subscription(Cog):
+    """Subscribing and Unsubscrbing to announcements and polls notifications."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    async def get_roles(self, ctx: Context) -> tuple:
+        """Returns a `tuple` with all the roles that you can subscribe/unsubscribe to."""
```

Comment:
I am not a fan of adding `Returns a 'tuple'` here, we can of course see that from the return type annotations.

We can do something like this:
```suggestion
        ""Gets the polls and announcements roles from the guild of the context."""
```